modules/running-insights-operator-gather-cli.adoc

```diff
@@ -35,8 +35,21 @@ spec:
 ----
 +
 --
-<1> Replace the `<your_data_gather>` with a unique name for your gather operation.
-<2> Enter individual gather operations to disable under the `gatherers` parameter. This example disables the `workloads` data gather operation and will run the remainder of the default operations. To run the complete list of default gather operations, leave the `spec` parameter empty. You can find the complete list of gather operations in the Insights Operator documentation.
+<1> Under *metadata*, replace `<your_data_gather>` with a unique name for the gather operation.
++
+[IMPORTANT]
+====
+Do not add a prefix of "periodic-gathering-" to the name of your gather operation because this string is reserved for other administrative operations and might impact the intended gather operation.
+====
++
```

Comment:
OK. https://github.com/openshift/openshift-docs/pull/81465/files seems to drop the plus here so maybe that will work?

Reply:
Done.